tests/unit/test_add_model.py

```diff
@@ -5,7 +5,7 @@
 
 def test_defaults(run: mocks.Run):
     run.handle(['juju', 'add-model', '--no-switch', 'new'])
-    juju = jubilant.Juju(model='initial')
+    juju = jubilant.Juju(model='initial', cli_version='3.6.9')
```

Comment:
What if instead of `cli_version` property we have an `Juju.is_juju_3` property and default it to `True`. That would get rid of all these.